src/components/App.tsx

```diff
@@ -3630,7 +3630,7 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
     const elements = this.scene.getElements();
     const element = this.getElementAtPosition(x, y);
-    const isMobile = getWhetherMobile();
+    const isMobile = getIsMobileMatcher().matches;
```

Comment:
But we still can (should) export `const isMobile => getIsMobileMatcher().matches` instead.